src/Controls/samples/Controls.Sample.Sandbox/Maui.Controls.Sample.Sandbox.csproj

```diff
@@ -10,7 +10,7 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <ApplicationTitle>.NET MAUI Controls Sandbox</ApplicationTitle>
+    <ApplicationTitle>Sandbox</ApplicationTitle>
```

Comment:
Right now my iPhone has 2 ".NETMAUICon..." apps that I never know which one I am using. We don't need this fancy name, we already know because we have a cool icon.